x-pack/plugins/canvas/common/functions/to.js

```diff
@@ -4,23 +4,34 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
 import { castProvider } from '../interpreter/cast';
 
 export const to = () => ({
   name: 'to',
   aliases: [],
-  help: 'Explicitly cast from one type to another.',
+  help: i18n.translate('xpack.canvas.functions.explicityCastFromTypeToTypeHelpText', {
+    defaultMessage: 'Explicitly cast from one type to another.',
+  }),
   context: {},
   args: {
     type: {
       types: ['string'],
-      help: 'A known type',
+      help: i18n.translate('xpack.canvas.functions.knownTypeHelpText', {
```

Comment:
knownTypeHelpText => to.typeHelpText

Reply:
Making this change here is not the approach we discussed. This function is being injected into a registry, the translation should be looked up at registry injection time, not here.